features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Tasks/TasksListing.js

```diff
@@ -65,10 +65,32 @@
            }
        );
    }
-   componentDidMount() {
 
+   //Since Same component is used for all the tasks, this is implemented to reload the tables
+   componentWillReceiveProps(nextProps){
```

Comment:
here I'm reusing the TaskListing component for all the tasks. componentDidMount() executes only for the first time it loads. since i'm reusing the same component for all the tasks, that component does not load again. only new properties are passed to that already loaded component. that's why need to use componentWillReceiveProps() method